src/components/login/WalletImportPage/WalletImportPage.js

```diff
@@ -22,6 +22,7 @@ import {
 import { SubmissionError } from 'redux-form'
 import profileService from '@chronobank/login/network/ProfileService'
 import { getAddress } from '@chronobank/core/redux/persistAccount/utils'
+import { navigateBack } from "../../../../packages/login-ui/redux/actions";
```

Comment:
import { navigateBack } from "@chronobank/login-ui/redux/actions"

Also do not use ";"